### PR TITLE
OZ-149:  Release of Ozone Docker

### PR DESCRIPTION
**Pull request by enyachoke**
Double commit to release Ozone Docker https://mekomsolutions.atlassian.net/browse/OZ-149

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -199,7 +199,7 @@ services:
 
   # OpenMRS 3 Frontend
   frontend:
-    image: openmrs/openmrs-reference-application-3-frontend:${O3_FRONTEND_TAG:-nightly}
+    image: openmrs/openmrs-reference-application-3-frontend:3.0.0-beta.8
```
Comment (Ruhanga) [inline docker-compose.yml]: We will need to add this `3.0.0-beta.8` appropriately to the quick start guide and the manual setup instructions as well.

**Review by Ruhanga — COMMENTED** [review]
Thanks @enyachoke, a quick one below.

File: docker-compose.yml (hunk 1 above)
Comment (enyachoke) [reply docker-compose.yml]: @Ruhanga  This is now resolved

**Review by rbuisson — APPROVED** [review]

**Comment by mks-d** [comment]
@rbuisson two things on this one:
1. @enyachoke you undid changes that I made to Docker File yesterday, you just overrode them.
2. Please try to respect our release commits naming conventions like we do with Maven - unless you bring a point that this is not relevant here.

**Pull request merged**